specs/experimental/standard-l2-genesis.md

```diff
@@ -290,9 +321,9 @@ via a deposit transaction from the `DEPOSITOR_ACCOUNT`.
 
 ##### `setIsthmus`
```

Comment:
This is fine to leave for now but for clarity, we will need to modify this, unclear which hardfork this will land in exactly right now